### PR TITLE
Add a file for Ankeny, IA

### PR DESCRIPTION
**Pull request by AdamPaolo**
:corn: :corn: :corn: :corn: :corn: :corn: :corn: :corn: This PR is for Ankeny. :corn: :corn: :corn: :corn: :corn: :corn: :corn: :corn: :corn:

1. How does Iowa make you feel?
2. Do you like cornfields?

### CODE REVIEW

**Comment by ghost** [comment]
1. Like I'm in Star Trek, but not the good parts.
2. Yes, as long as I'm not lost in them.

File: AdamPaolo-Ankeny.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Ankeny, IA
+
+### Restaurants
+
+### Places the locals like
+
```
Comment (ghost) [inline AdamPaolo-Ankeny.md:5]: Is this a commentary on Iowa?

Comment (AdamPaolo) [reply AdamPaolo-Ankeny.md:5]: ... Yes,

**Comment by AdamPaolo** [comment]
This pull request resolves #11

**Review by ghost — APPROVED** [review]

**Pull request merged**